docs/source/about/advanced_usage.rst

```diff
@@ -1123,6 +1123,39 @@ vt_coder
 
       vt_coder = auto
 
+mf_rc
+^^^^^
+
+**Description**
+   The encoder rate control.
+
+   .. Note:: This option only applies when using mediafoundation `encoder`_.
```

Comment:
Do we need to add a new choice to the `encoder` choices table?

Reply:
Added